Source/Fuse.Common/FastMatrix.uno

```diff
@@ -19,6 +19,19 @@ namespace Fuse
 		*/
 		public bool IsValid { get { return _isValid; } }
 
+		public bool IsIdentity
+		{
+			get
+			{
+				if (!_hasNonTranslation)
+					return _matrix.M41M42M43 == float3(0);
+				else
+				{
+					return _matrix == float4x4.Identity;
```

Comment:
I wonder, given how CLR isn't inlining stuff, if manually inlining this here might be helpful?